frontend/app/src/shared/components/form/utils/getFormFieldFromRelationship.ts

```diff
@@ -11,21 +11,27 @@ import { getRelationshipParent } from "@/shared/components/form/utils/getRelatio
 import { isFieldDisabled } from "@/shared/components/form/utils/isFieldDisabled";
 import { isMaxCount, isMinCount, isRequired } from "@/shared/components/form/utils/validation";
 
+interface GetFormFieldsProps {
```

Comment:
Props -> Params. Props is use for components, params for function.
GetFormFields->GetFormFieldFromRelationship